test/feature-benchmark/mzcompose.py

```diff
@@ -105,6 +106,17 @@ def make_comparator(name: str, type: MeasurementType) -> Comparator:
     Minio(setup_materialize=True),
     KgenService(),
     Postgres(),
+    MySql(
+        additional_args=[
```

Comment:
Maybe we should set these args as default since we'll be using them in every test?

Reply:
yeah that's probably a good idea -- I've repeated these same arguments in a few places already!

Reply:
✅